pkg/unikontainers/unikernels/unikernel.go

```diff
@@ -33,29 +36,21 @@ type UnikernelParams struct {
 
 var ErrNotSupportedUnikernel = errors.New("unikernel is not supported")
 
-func UnikernelCommand(unikernelType UnikernelType, data UnikernelParams) (string, error) {
+func New(unikernelType UnikernelType) (Unikernel, error) {
 	switch unikernelType {
 	case RumprunUnikernel:
-		unikernel, err := newRumprun(data)
+		unikernel, err := newRumprun()
```

Comment:
As mentioned before, the error returned by newRumprun() will always be nil, so it could be omitted.

Reply:
I will fix it.